src/kano_safe_mode_services/kano_safe_mode_send_logs.py

```diff
@@ -0,0 +1,54 @@
+# kano_safe_mode_send_logs.py
+#
+# Copyright (C) 2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+#
+# TODO
+
+
+import os
+import time
+
+from kano.utils.shell import run_cmd
+from kano.logging import logger
+
+from kano_safe_mode_services.paths import LOGS_FLAG_PATH, LOGS_PATH, \
+    TMP_LOGS_OUTPUT_PATH
+
+
+def _logs_already_sent():
+    """TODO"""
+    return os.path.exists(LOGS_FLAG_PATH)
+
+
+def main():
+    # Ensure script is not run as root.
+    if os.getuid() == 0:
+        return 10
+
+    if _logs_already_sent():
+        return
+
+    if not os.path.exists(LOGS_PATH):
```

Comment:
There is a catch here if a user:
 - had previously gone into safe mode and already have a `/boot/support_logs.tgz`
 - update to v4.2
this service will then send us the logs as if they've just gone into Safe Mode.

Changing the filename will likely create a second `.tgz` in `/boot/` and free space is precious there. We could add a post-update scenario to remove any existing logs if we don't want them.